docs/blog/rendered/campaign-finance.ipynb

```diff
@@ -0,0 +1,2176 @@
+{
```

Comment:
<p>We do have <code>bucket</code> and <code>histogram</code> methods, but I suspect they aren't working for you. We should definitely see if we can make them work for your use cases!  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ibis-project/ibis/pull/5727/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Reply:
<p>hmmm, I had found <code>histogram()</code>,and indeed it wasn't working. But I hadn't found <code>bucket()</code>, let me try that    </p>

Reply:
<p>.bucket() works, I migrated to that.</p>